<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PokemonClass</class>
 <widget class="QMainWindow" name="PokemonClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pokemon</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1001</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Pokemom/Resources/background.jpg); 
background-repeat: no-repeat; 
background-position: center; 
background-attachment: fixed; 
background-size: cover;</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1001</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
background-image: none;
</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_main">
     <widget class="QLabel" name="lbl_game">
      <property name="geometry">
       <rect>
        <x>519</x>
        <y>419</y>
        <width>47</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_pokemon">
      <property name="geometry">
       <rect>
        <x>519</x>
        <y>335</y>
        <width>47</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_test_case">
      <property name="geometry">
       <rect>
        <x>519</x>
        <y>293</y>
        <width>47</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_test_case">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>290</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:none;
background-color:white;</string>
      </property>
      <property name="text">
       <string>Test Case</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_game">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>410</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:none;
background-color:white;</string>
      </property>
      <property name="text">
       <string>Game</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_move">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>370</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:none;
background-color:white;</string>
      </property>
      <property name="text">
       <string>Move</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_pokemon">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>330</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:none;
background-color:white;</string>
      </property>
      <property name="text">
       <string>Pokemon</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_battle">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>450</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:none;
background-color:white;</string>
      </property>
      <property name="text">
       <string>Battle</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_move">
      <property name="geometry">
       <rect>
        <x>519</x>
        <y>377</y>
        <width>47</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>631</width>
        <height>221</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"> background-image: url(:/Pokemom/Resources/pokemon-logo.png);
    background-repeat: no-repeat;
    background-position: center;
background-attachment: fixed;
    background-size: cover;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_battle">
     <widget class="QPushButton" name="pb_battle_battle">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>811</x>
        <y>341</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:none;
background-color:white;</string>
      </property>
      <property name="text">
       <string>Battle</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_battle_pokemon">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>811</x>
        <y>374</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:none;
background-color:white;</string>
      </property>
      <property name="text">
       <string>Pokemon</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_battle_bag">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>811</x>
        <y>407</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:none;
background-color:white;</string>
      </property>
      <property name="text">
       <string>Bag</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_battle_run">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>811</x>
        <y>440</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:none;
background-color:white;</string>
      </property>
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="text_edit_battle_content">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>361</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_battle_back">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>440</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:none;
background-color:white;</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_battle_player_pokemon">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>390</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Player Pokemon</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_battle_opposite_pokemon">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>260</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Opposite Pokemon</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_battle_player_pokemon_blood">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>420</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Player Blood</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_battle_opposite_pokemon_blood">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>280</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Opposite Blood</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>998</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Pokemon.qrc"/>
 </resources>
 <connections/>
</ui>
